src/session.rs

```diff
@@ -164,8 +164,8 @@ impl Session {
             0x9  => self.stream.lock().unwrap().handle(cmd, data),
             0xd | 0xe => self.audio_key.lock().unwrap().handle(cmd, data),
             0xb2...0xb6 => self.mercury.lock().unwrap().handle(cmd, data),
-            0xac => eprintln!("Authentication succeedded"),
-            0xad => eprintln!("Authentication failed"),
+            0xac => info!("Authentication succeedded"),
```

Comment:
Not related to this patch, but should `login()` perhaps call `poll()` itself before returning (instead of session clients doing it), and these two values map to some internal state that `login()` uses to return Ok or Err so that clients can control the UI of what auth success/failure looks like?